src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyForInitializerPadCheckTest.java

```diff
@@ -91,7 +92,8 @@ public void testPadOptionValueOf() {
         assertEquals(PadOption.NOSPACE, option, "Result of valueOf is invalid");
     }
 
-    /* Additional test for jacoco, since valueOf()
+    /**
+     * Additional test for jacoco, since valueOf()
```

Comment:
how this changes are related with this check?

Reply:
Initially, this comment was creating a violation, so to fix it I made this change. Then later to improve the check I added a method `isJavadocComment` to only validate the comments if they are Javadoc comments. Hence now this won't create a violation. I'll revert this change.

Reply:
Done

Reply:
these changes are still here. I suggest to create different PR for corrections for this check to avoid confusion and leave here only changes necessary for new check